x-pack/plugins/remote_clusters/public/sections/remote_cluster_add/remote_cluster_form.js

```diff
@@ -126,7 +139,45 @@ export class RemoteClusterFormUi extends Component {
     return null;
   }
 
+  getLocalSeedErrors = (seedNode) => {
+    const { intl } = this.props;
+
+    const errors = [];
+
+    const isInvalid = !isSeedNodeValid(seedNode);
+
+    if (isInvalid) {
+      errors.push(intl.formatMessage({
+        id: 'xpack.remoteClusters.remoteClusterForm.localSeedError.invalidCharactersMessage',
+        defaultMessage: `Seed nodes must consist of valid characters, optionally separated by
```

Comment:
`optionally separated by periods` could be `with optional parts separated by periods`